tests/standard_to_enhanced/navjoy_translator_test.py

```diff
@@ -52,27 +52,28 @@ def test_parse_reduction_zone_linestring(mock_dt, mock_dt_3):
     test_feature = navjoy_translator.parse_reduction_zone(standard)
 
     expected_feature = {
```

Comment:
Pull to separate file like in icone_translator_test.py?

Reply:
Moved to a separate file